pkg/reconciler/configuration/configuration_test.go

```diff
@@ -39,8 +37,8 @@ import (
 	"knative.dev/serving/pkg/apis/serving/v1alpha1"
 	"knative.dev/serving/pkg/apis/serving/v1beta1"
 	"knative.dev/serving/pkg/gc"
+	"knative.dev/serving/pkg/reconciler/gc/config"
```

Comment:
I'm surprised we need the GC config anywhere in the configuration controller anymore.

Reply:
Ahh I forgot to remove it. Just updated the PR.